### PR TITLE
Added scripts for 2016-11-29-volunteer-hoc-is-here mailing

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

File: bin/mail/2016-11-29-volunteer-hoc-is-here/generate-list [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# VolunteerEngineerSubmission2015
+volunteers = query_subscribed_contacts(q: 'kind_s:VolunteerEngineerSubmission2015')
```
Comment (ashercodeorg) [inline bin/mail/2016-11-29-volunteer-hoc-is-here/generate-list:5]: Presumably we are intentionally not including volunteers for prior years?

**Comment by aoby** [comment]
RE: staging-next, yes that's intentional since staging is locked down and I'll have to manually copy the scripts and email template to production-daemon to send these today anyway. I'll merge this to staging-next to preserve the history.

**Comment by ashercodeorg** [comment]
IMO, if we are sending the email from production, we should merge this to production (via staging). But as you wish...

File: bin/mail/2016-11-29-volunteer-hoc-is-here/generate-list (hunk 1 above)
Comment (aoby) [reply bin/mail/2016-11-29-volunteer-hoc-is-here/generate-list:5]: Yes, as per @tanyaparker

**Comment by aoby** [comment]
We often make changes to the email templates and sometimes the scripts last-minute, and don't want or need to wait for a full deployment since it doesn't affect the front ends. I agree it would be ideal to merge the latest scripts and templates into the production branch if that didn't trigger a full deployment, but these are tightly coupled.

We could merge it into staging rather than staging-next and scoop up the email templates so it goes into prod on the next DTP but we still probably want to send this email out of band (before the DTP). Would you prefer that? 

What do you think, @tanyaparker?

**Comment by ashercodeorg** [comment]
Yes, feel welcome to send from production before this stuff gets merged. But I'd rather see this go to staging then staging-next.

**Comment by aoby** [comment]
Ok, that makes sense. I'll rebase against staging...

**Pull request merged**